### DISCUSSION TITLE
radar graphic

### DISCUSSION THREAD

**Discussion by Oassinde** [community/community#67834] [Programming Help]
### Select Topic Area

Question

### Body

how can i do some radar graphical with 10 caracreristical and 40 genotipos?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Oassinde ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
